docs/codacy-api/examples/obtaining-audit-log-events-for-organization.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Example of how to obtain audit log events for an organization using the Codacy API endpoint listAuditLogsForOrganization.
+---
+
+# Obtaining audit log events for an organization
```

Comment:
@franciscoovazevedo @geirolz Let me know if this example makes sense. 
We are already providing some [similar examples](https://docs.codacy.com/codacy-api/examples/obtaining-code-quality-metrics-for-files/) for other endpoints.

Reply:
I removed the example, as it doesn't add much value.